crates/router/src/connector/paybox/transformers.rs

```diff
@@ -338,23 +379,49 @@ impl TryFrom<&PayboxRouterData<&types::PaymentsAuthorizeRouterData>> for PayboxP
                     DateFormat::DDMMYYYYHHmmss,
                 )
                 .change_context(errors::ConnectorError::RequestEncodingFailed)?;
-
-                Ok(Self {
-                    date: format_time.clone(),
-                    transaction_type,
-                    paybox_request_number: get_paybox_request_number()?,
-                    amount: item.amount,
-                    description_reference: item.router_data.connector_request_reference_id.clone(),
-                    version: VERSION_PAYBOX.to_string(),
-                    currency,
-                    card_number: req_card.card_number,
-                    expiration_date,
-                    cvv: req_card.card_cvc,
-                    activity: PAY_ORIGIN_INTERNET.to_string(),
-                    site: auth_data.site,
-                    rank: auth_data.rang,
-                    key: auth_data.cle,
-                })
+                if item.router_data.is_three_ds() {
```

Comment:
Also test Capture, Psync and Refund Flow for 3DS transactions.

Reply:
tested , Working